src/View/Home/index.html.twig

```diff
@@ -1,6 +1,74 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    <h1>Hello Wilder !</h1>
-    <p>Find a complete CRUD example on <a href="/items">/items</a> page.</p>
+	<div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="false">
```

Comment:
Please start every page component with a  tag, for referencing features

Reply:
add tag section/ for carrousel and article/ for display last articles